pkg/telemetry/analyticsservice.go

```diff
@@ -28,14 +31,17 @@ import (
 type AnalyticsService interface {
 	SendStats(ctx context.Context, stats []*livekit.AnalyticsStat)
 	SendEvent(ctx context.Context, events *livekit.AnalyticsEvent)
+	SendNodeRoomStates(ctx context.Context, nodeRooms *livekit.AnalyticsNodeRooms)
 }
 
 type analyticsService struct {
-	analyticsKey string
-	nodeID       string
+	analyticsKey   string
+	nodeID         string
+	sequenceNumber atomic.Uint64
 
-	events livekit.AnalyticsRecorderService_IngestEventsClient
-	stats  livekit.AnalyticsRecorderService_IngestStatsClient
+	events    livekit.AnalyticsRecorderService_IngestEventsClient
+	stats     livekit.AnalyticsRecorderService_IngestStatsClient
+	nodeRooms livekit.AnalyticsRecorderService_IngestNodeRoomStatesClient
```

Comment:
just curious, why do these need different clients?

Reply:
and I don't see them initialized.. are there API to set them elsewhere?

Reply:
these are streaming apis, so each get their own client